polymer-cookie.html

```diff
@@ -116,13 +125,21 @@
           type: Boolean,
           value: false,
           reflectToAttribute: true
+        },
+        autoUnset: {
```

Comment:
Please provide a properties description for consistency with the others and leave a blank space between the previous one.